src/rgw/rgw_admin.cc

```diff
@@ -4693,6 +4704,39 @@ int main(int argc, char **argv)
     }
   }
 
+  if (opt_cmd == OPT_LC_LIST) {     
+    formatter->open_array_section("life cycle progress");
+    map<string, int> bucket_lc_map;
+    string marker;
+#define MAX_LC_LIST_ENTRIES 100
```

Comment:
@theanalyst 
you can adjust system time after set lifecycle to a bucket to invoke ,or exec radosgw-admin lc process manully. but It only do once per day.